release.py

```diff
@@ -0,0 +1,848 @@
+
+        if not suggested_version:
+            logger.error('Error: No version specified and no suggestion available')
+            continue
+
+        if not re.match(r'^\d+\.\d+\.\d+$', suggested_version):
+            logger.error('Error: Version must be in format X.Y.Z')
+            continue
+
+        confirm = (
+            input(f"Confirm using scenario version '{suggested_version}'? [y/N]: ").strip().lower()
+        )
+
+        if confirm == 'y':
+            break
+
+        logger.info("Let's try again...")
+
+    return suggested_version
+
+
+def update_versions_for_release(tag: str):
+    """Update version files to the specified release version."""
+    scenario_version = get_new_scenario_version_for_release()
+    update_ops_version(tag, scenario_version)
+    update_testing_version(tag, scenario_version)
+    update_tracing_version(tag)
+    update_uv_lock()
+
+
+def get_new_version_post_release(repo: github.Repository.Repository, branch_name: str) -> str:
+    """Get the new version after the release.
+
+    Default value is generated by bumping the minor version of the latest release's tag
+    and adding the '.dev0' suffix.
+
+    Can be overridden with a user-provided version.
+    """
+    latest_version = get_latest_release_tag(repo, branch_name)
+
+    if latest_version is None:
+        logger.info('No version tags found in branch "{branch_name}".')
+        suggested_version = ''
+    else:
+        logger.info(f'Latest version in branch "{branch_name}": {latest_version}')
+
+        # Check if the latest version is in SEMVER, in case it has a suffix.
+        if not re.match(r'^\d+\.\d+\.\d+$', latest_version):
+            logger.warning(
+                f'Latest version "{latest_version}" must be in format '
+                f'X.Y.Z. Please input the new version manually (remember to add the .dev0 suffix).'
+            )
+            suggested_version = ''
+        else:
+            suggested_version = bump_minor_version(latest_version) + '.dev0'
+            logger.info(f'Suggested new version: {suggested_version}')
+
+    while True:
+        user_input = input(
+            f'Input post release version (press enter to use suggested version '
+            f"{suggested_version or 'required'}), or 'c' to cancel: "
+        ).strip()
+
+        if user_input.lower() == 'c':
+            logger.warning('Post release canceled.')
+            exit(0)
+
+        new_version = user_input if user_input else suggested_version
+
+        if not new_version:
+            logger.error('Error: No version specified and no suggestion available')
+            continue
+
+        confirm = (
+            input(f"Confirm using version '{new_version}' for post release'? [y/N]: ")
+            .strip()
+            .lower()
+        )
+
+        if confirm == 'y':
+            break
+
+        logger.info("Let's try again...")
+
+    return new_version
+
+
+def get_new_scenario_version_post_release() -> str:
```

Comment:
I would be fine if we removed this and made it `{ops_major+5}.{ops_minor}.{ops.bugfix}`.

Or if we are ok with ops-scenario bumping a major version without ops, then I think we should handle that for ops-tracing too. But I think we are sticking with no major version changes for now?

Reply:
If there is no major version change, it is fine to use a suggested version, either from `{ops_major+5}.{ops_minor}.{ops.bugfix}`, or parse what's in the `testing/pyproject.toml` file and bump the minor version (part of the logic in this function).

Since this script was created during the 2.x -> 3 release, I added an optional override. There will still be an automatically suggested version in case you don't want to input anything, so I am keeping this logic for now.